packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCard/AddCard.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { Image, ImageFit } from 'office-ui-fabric-react/lib/Image';
+import { getStyles } from './AddCard.styles';
+import { IAddCardProps, IAddCardStyles } from './AddCard.types';
+
+export class AddCard extends React.Component<IAddCardProps> {
+  constructor(props: IAddCardProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const getClassNames = classNamesFunction<IAddCardProps, IAddCardStyles>();
+    const classNames = getClassNames(getStyles!);
+    const { title, imageSrc, description, id } = this.props;
+    let imgClassName = classNames.imageWrapper;
+    let textContainerClassName = classNames.textContainer;
+    let addIconClassName = classNames.icon;
+    if (id) {
+      imgClassName = 'addCardImg' + id + ' ' + imgClassName;
```

Comment:
For your consideration, OUFR has a utility function `getId` for generating unique identifiers with a prefix located in https://github.com/OfficeDev/office-ui-fabric-react/blob/72942a6152855a0717f6bcdb0f12309c5be5d25b/packages/utilities/src/object.ts#L94